src/apps/relay/mainrelay.c

```diff
@@ -2632,15 +2628,12 @@ int main(int argc, char **argv)
 
 ////////// OpenSSL locking ////////////////////////////////////////
 
-#if defined(OPENSSL_THREADS)
```

Comment:
`OPENSSL_THREADS` only makes sense in 1.0.2 and below
aything above that do not need locking mechanisms